actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -22,7 +22,7 @@ module FormTagHelper
       mattr_accessor :embed_authenticity_token_in_remote_forms
       self.embed_authenticity_token_in_remote_forms = nil
 
-      # Starts a form tag that points the action to a url configured with <tt>url_for_options</tt> just like
+      # Starts a form tag that points the action to a url configured with +url_for_options+ just like
```

Comment:
Is this necessary?

Reply:
Thanks, It was not needless change. I've restored it.